libs/libvtrcapnproto/connection_map.capnp

```diff
@@ -5,16 +5,20 @@ using Matrix = import "matrix.capnp";
 struct VprCostEntry {
     delay @0 :Float32;
     congestion @1 :Float32;
-    fill @2 :Bool;
```

Comment:
Please keep the `fill` field, as it is useful for debugging.  I'm fine if we move it around in memory for performance reasons.

Reply:
I added this back and the logic behind it.